test/components/views/rooms/wysiwyg_composer/components/WysiwygComposer-test.tsx

```diff
@@ -427,7 +427,7 @@ describe("WysiwygComposer", () => {
         });
         it("typing a space to trigger an emoji replacement", async () => {
             fireEvent.input(screen.getByRole("textbox"), {
-                data: ":P",
+                data: ":-P",
```

Comment:
This implies that `:P` no longer works?

Reply:
I was testing this locally, I didn't mean to push up this change. Will add as a second case. Thanks for pointing it out.